addons/website_event_track/static/src/js/website_event_track_proposal_add_tags.js

```diff
@@ -0,0 +1,106 @@
+odoo.define('website_event_track.website_event_track_proposal_add_tags', function (require) {
+    'use strict';
+
+    var core = require('web.core');
+    var publicWidget = require('web.public.widget');
+
+    var _t = core._t;
+
+    publicWidget.registry.websiteEventProposalTags = publicWidget.Widget.extend({
+
+    selector: '.o_wetrack_proposal_add_tags_widget',
+
+    init: function (parent, options){
+        this.tagIds = [];
+        this._super(parent, options);
+    },
+
+    start: function () {
+        var self = this;
+        return this._super.apply(this, arguments).then(function () {
+            self._bindSelect2Dropdown();
+        });
+    },
+
+    /**
+     * Handler for select2 on tags added to the proposal track form.
+     *
+     * @private
+     */
+    _bindSelect2Dropdown: function () {
+        var self = this;
+        this.$('.o_wetrack_select2_tags').select2(this._select2Wrapper(_t('Select categories'),
+            function () {
+                return self._rpc({
+                    route: "/event/track_proposal/fetch_tags",
+                    params: {
+                        fields: ['name', 'category_id'],
+                        domain: [['id', 'not in', self.tagIds],['color', '!=', 0]],
```

Comment:
Well normally ACLs are now fixed and public / portal cannot see uncolored tags. Can't we remove this domain part ? If an event manager want to use a colorless tag from frontend, we should not prevent him from doing it. Maybe you can rebase on latest master and check it works out of the box.

Reply:
Indeed! part removed